corehq/apps/couch_sql_migration/couchsqlmigration.py

```diff
@@ -753,6 +802,8 @@ def try_obj(self, lock_ids, queue_obj):
 
         Returns :boolean: True if it acquired the lock, False if it was added to queue
         """
+        if not lock_ids:
+            self._add_item(lock_ids, queue_obj, to_queue=False)
```

Comment:
should this `return True` afterwards?

Reply:
Ah yeah. 001562d

Reading through the code I'm realizing that tho this short circuits the process and arguably adds some clarity about the no lock_ids case, it isn't necessary and just adds some difficulty when reading it. I'm leaning towards keeping it in, but general thoughts on whether a block like that is good practice in general?